tests/unit/plugins/inventory/test_kubevirt.py

```diff
@@ -155,7 +155,7 @@
     }
 )
 LOADBALANCER_SERVICE_WITHOUT_SELECTOR_AND_SSH_PORT = merge_dicts(
```

Comment:
Either use `LOADBALANCER` or `LB` everywhere?

Reply:
Let's stick with the `LOADBALANCER` option.